docs/guide/introduction.md

```diff
@@ -0,0 +1,62 @@
+# Introduction
+
+`asdf` is a tool version manager. All tool version definitions are contained within one file (`.tool-versions`) which you can check-in to your project's Git repository to share with your team, ensuring everyone is using the **exact** same versions of tools.
+
+`asdf` was built to address two core issues:
+
+1. different CLI tools to manage each runtime/tool in a repository/project
+2. different CLI commands & configuration files (`.*-version` files) per manager for each of the runtime/tools
```

Comment:
To me this is just one issue. If I had to sum this up I'd say something like this:

The old way of working required multiple CLI version managers, each with their distinct API, implementation (e.g. `$PATH` manipulation, shims, environment variables, etc...) and configuration files. asdf provides a single interface and configuration file to simplify development workflows, and can be extended to all tools and runtimes via a simple plugin interface.

What do you think?

Reply:
That's much better, thanks. I found this introduction page the most difficult to write.

Reply:
I updated with your wording but with the "implementation" and parens as the last item in the first sentence.

Reply:
> That's much better, thanks. I found this introduction page the most difficult to write.

Yeah, I've spent too much time on asdf to be good at describing it to people who aren't familiar with it. This introduce page looks great thought!